### PR TITLE
docker-build, yum: add Indonesian translation

### PR DESCRIPTION
**Pull request by fadharpra**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fadharpra** [comment]
Hi @reinhart1010, kindly review my request

File: pages.id/linux/yum.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# yum
+
+> Utilitas manajemen paket untuk RHEL, Fedora, dan CentOS (untuk versi-versi yang lebih lama).
+> Untuk perintah-perintah setara dalam pengelola paket lainnya, lihat https://wiki.archlinux.org/title/Pacman/Rosetta.
+> Informasi lebih lanjut: https://manned.org/yum.
+
+- Menginstall sebuah paket baru
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:7]:
```suggestion
- Instal sebuah paket baru
```
Imperative mood.

File: pages.id/linux/yum.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`yum install {{nama_paket}}`
+
+- Menginstall sebuah paket baru dan mengasumsikan jawaban ya untuk semua pertanyaan (juga berfungsi dengan perintah pembaruan, sangat berguna untuk pembaruan otomatis):
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:11]:
```suggestion
- Instal sebuah paket baru dan mengasumsikan jawaban [y]a untuk semua pertanyaan (juga berfungsi dengan perintah pembaruan, sangat berguna untuk pembaruan otomatis):
```

File: pages.id/linux/yum.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`yum -y install {{nama_paket}}`
+
+- Mencari sebuah paket yang menyediakan suatu perintah tertentu:
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:15]:
```suggestion
- Cari sebuah paket yang menyediakan suatu perintah tertentu:
```

File: pages.id/linux/yum.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`yum provides {{perintah}}`
+
+- Menghapus sebuah paket
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:19]:
```suggestion
- Hapus paket yang terpasang sebelumnya:
```

File: pages.id/linux/yum.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`yum remove {{paket}}`
+
+- Menampilkan pembaruan yang tersedia untuk paket yang terpasang:
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:23]:
```suggestion
- Tampilkan pembaruan yang tersedia untuk paket yang terpasang:
```

File: pages.id/linux/yum.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+`yum check-update`
+
+- Memperbarui seluruh paket yang terpasang ke versi terbaru:
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:27]:
```suggestion
- Perbarui seluruh paket yang terpasang ke versi terbaru:
```

File: pages.id/linux/yum.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# yum
+
+> Utilitas manajemen paket untuk RHEL, Fedora, dan CentOS (untuk versi-versi yang lebih lama).
+> Untuk perintah-perintah setara dalam pengelola paket lainnya, lihat https://wiki.archlinux.org/title/Pacman/Rosetta.
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:4]:
```suggestion
> Untuk perintah-perintah setara dalam pengelola paket lainnya, lihat <https://wiki.archlinux.org/title/Pacman/Rosetta>.
```
Use angled brackets for links (useful for some Markdown parsers like Kramdown)

File: pages.id/linux/yum.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+> Informasi lebih lanjut: https://manned.org/yum.
```
Comment (reinhart1010) [inline pages.id/linux/yum.md:5]:
```suggestion
> Informasi lebih lanjut: <https://manned.org/yum>.
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, here are some comments from mine.

**Comment by fadharpra** [comment]
Hi @reinhart1010, i've update the file. Kindly check again ya

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

**Pull request merged**